tests/ActivityLoggerTest.php

```diff
@@ -195,6 +195,40 @@ public function it_will_use_the_logged_in_user_as_the_causer_by_default()
         $this->assertEquals($userId, $this->getLastActivity()->causer->id);
     }
 
+    /**
+     * @test
+     *
+     * @requires !Travis
+     */
+    public function it_can_log_activity_using_an_anonymous_causer()
+    {
+        activity()
+            ->causedByAnonymous()
+            ->log('hello poetsvrouwman');
+
+        $this->assertEquals(null, $this->getLastActivity()->causer_id);
```

Comment:
Please use `assertNull()` instead of `assertEquals(null)`.

Reply:
Sure thing, will adjust that

Reply:
https://github.com/spatie/laravel-activitylog/pull/605/commits/21c115655066cb7c1febfaaf01086701c5bd2600